apps/aecore/lib/aecore/tx/signed_tx.ex

```diff
@@ -182,9 +182,14 @@ defmodule Aecore.Tx.SignedTx do
         %{"data" => DataTx.serialize(tx.data), "signature" => signature_serialized}
 
       _ ->
+        serialized_signatures =
+          for signature <- tx.signatures do
+            Serialization.serialize_value(signature, :signature)
+          end
+
         %{
           "data" => DataTx.serialize(tx.data),
-          "signature" => Serialization.serialize_value(:signature)
+          "signatures" => serialized_signatures
```

Comment:
is the name changed on purpose?

Reply:
Yes, because here, we are matching this clause whenever the signatures length is > 1 which means that we've got a list of signatures here.